receiver/mongodbreceiver/metadata.yaml

```
@@ -1 +1,125 @@
 name: mongodbreceiver
+
+attributes:
+  database:
+    description: The name of a database.
+  memory_type:
+    description: The type of memory used.
+    enum:
+      - resident
+      - virtual
+      - mapped
+      - mappedWithJournal
+  operation:
+    description: The mongoDB operation being counted.
+    enum:
+      - insert
+      - query
+      - update
+      - delete
+      - getmore
+      - command
+  connection_type:
+    description: The status of the connection.
+    enum:
+      - active
+      - available
+      - current
+  type:
+    description: The result of a cache request.
+    enum:
+      - hit
+      - miss
+
+metrics:
+  mongodb.cache.operations:
+    description: The number of cache operations of the instance.
+    unit: 1{.operations}
+    enabled: true
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+    attributes: [type]
+  mongodb.collections:
+    description: The number of collections.
+    unit: 1
+    enabled: true
+    gauge:
+      value_type: double
+    attributes: [database]
+  mongodb.data.size:
+    description: The size in bytes of the collection. Data compression does not affect this value.
+    unit: By
+    enabled: true
+    gauge:
+      value_type: double
+    attributes: [database]
+  mongodb.connections:
+    description: The number of connections. 
+    unit: 1
+    enabled: true
+    gauge:
+      value_type: double
+    attributes: [database, connection_type]
+  mongodb.extents:
+    description: The number of extents.
+    unit: 1
+    enabled: true
+    gauge:
+      value_type: double
+    attributes: [database]
+  mongodb.global_lock.hold:
```

```suggestion
  mongodb.global_lock.time:
```

This is a [time metric](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-naming).